src/Nordea/Css.elm

```diff
@@ -13,16 +34,181 @@ module Nordea.Css exposing
+-}
+gridRow : String -> Css.Style
+gridRow gridRowLines =
+    Css.property "grid-row" gridRowLines
+
+
+{-| Sets [`grid-row-start`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-row-start).
+-}
+gridRowStart : String -> Css.Style
+gridRowStart gridRowLine =
+    Css.property "grid-row-start" gridRowLine
+
+
+{-| Sets [`grid-row-end`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-row-end).
+-}
+gridRowEnd : String -> Css.Style
+gridRowEnd gridRowLine =
+    Css.property "grid-row-end" gridRowLine
+
+
+{-| Sets [`grid-template-areas`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas).
+
+For instance,
+
+    gridTemplateAreas ["header header", "nav main"]`
+
+Becomes
+
+    grid-template-areas:
+        "header header"
+        "nav    main";
+
+-}
+gridTemplateAreas : List String -> Css.Style
+gridTemplateAreas areas =
+    Css.property "grid-template-areas" (List.map (\row -> "\"" ++ row ++ "\"") areas |> String.join " ")
+
+
+{-| Sets [`grid-template-columns`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns).
+-}
+gridTemplateColumns : String -> Css.Style
+gridTemplateColumns columns =
+    Css.property "grid-template-columns" columns
+
+
+{-| Sets [`grid-template-rows`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-rows).
+-}
+gridTemplateRows : String -> Css.Style
+gridTemplateRows rows =
+    Css.property "grid-template-rows" rows
+
+
+{-| Sets [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap).
+-}
 gap : LengthOrNoneOrMinMaxDimension compatible -> Css.Style
 gap argA =
     Css.property "gap" argA.value
 
 
+{-| Sets [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap).
+-}
 gap2 : LengthOrNoneOrMinMaxDimension compatible -> LengthOrNoneOrMinMaxDimension compatible2 -> Css.Style
 gap2 argA argB =
     Css.property "gap" (argA.value ++ " " ++ argB.value)
 
 
+{-| Sets [`column-gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap).
+-}
+columnGap : LengthOrNoneOrMinMaxDimension compatible -> Css.Style
+columnGap argA =
+    Css.property "column-gap" argA.value
+
+
+{-| Sets [`row-gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap).
+-}
+rowGap : LengthOrNoneOrMinMaxDimension compatible -> Css.Style
+rowGap argA =
+    Css.property "row-gap" argA.value
+
+
+{-| Sets [`align-content`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content).
+-}
+alignContent : String -> Css.Style
+alignContent alignment =
+    Css.property "align-content" alignment
+
+
+{-| Sets [`justify-self`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-self).
+-}
+justifySelf : String -> Css.Style
+justifySelf alignment =
+    Css.property "justifySelf" alignment
+
+
+{-| Sets [`justify-items`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items).
+-}
+justifyItems : String -> Css.Style
+justifyItems alignment =
+    Css.property "justifyItems" alignment
+
```

Comment:
Do these support the same values as alignSelf, justifyContents etc? if so, use `(ExplicitLength IncompatibleUnits -> Style)` as the param type?